.eslintrc.js

```diff
@@ -110,44 +119,19 @@ module.exports = {
       'extends': ['plugin:jest/recommended', 'plugin:jest/style'],
     },
     {
-      'files': ['test/**/*.test.js'],
+      'files': ['test/**/*.test.js', 'test/**/*.test.mjs'],
       'rules': {
+        'import/namespace': 'off',
         'jest/expect-expect': 'off',
         'jest/no-conditional-expect': 'off',
         'jest/no-done-callback': 'off',
-        'jest/no-standalone-expect': 'off',
-        'jest/no-try-expect': 'off',
       },
     },
     {
-      'files': ['test/**/cloudant.v1.test.js'],
+      'files': ['test/**/cloudant.*.test.js', 'test/**/auth-helper.js'],
       'rules': {
+        '@typescript-eslint/no-require-imports': 'off',
         'prettier/prettier': 'off',
-        'object-shorthand': 'off',
-      },
-    },
-    {
-      'files': [
-        'test/**/readme.integration.test.js',
-        'test/examples/src/ts/*.ts',
-      ],
-      'rules': {
-        'no-console': 'off',
-      },
-    },
-    {
-      'files': ['test/examples/src/js/*.js'],
-      'rules': {
-        'dot-notation': 'off',
-      },
-    },
-    {
-      'files': [
-        'test/examples/src/js/CreateDbAndDoc.js',
-        'test/examples/src/ts/CreateDbAndDoc.ts',
-      ],
-      'rules': {
-        'prefer-template': 'off',
```

Comment:
Could you turn file extensions of `.js` into `.mjs` instead of removing this block?

Reply:
As you can see they all are 'off' overrides and linter is passing without them, so this block been kind of meaningless.